src/SphericalHarmonics.jl

```diff
@@ -92,9 +96,27 @@ getindex(S::AbstractSphericalHarmonic, x::StaticVector{3}, k::Int) = S[x, findbl
 # Expansion
 ##
 
-# const FiniteSphericalHarmonic{T} = SubQuasiArray{T,2,SphericalHarmonic{T},<:Tuple{<:Inclusion,<:OneTo}}
+const FiniteSphericalHarmonic{T} = SubQuasiArray{T,2,SphericalHarmonic{T},<:Tuple{<:Inclusion,<:BlockSlice{BlockRange1{OneTo{Int}}}}}
+
+function grid(S::FiniteSphericalHarmonic)
+    T = real(eltype(S))
+    N = blocksize(S,2)
```

Comment:
Are you overloading blocksize(A,d) or blockaxes() somewhere else, because this line is causing the tests for your transforms to fail on my end?

```julia
julia> N = 2
2

julia> S = SphericalHarmonic()[:,Block.(Base.OneTo(N))]
view(SphericalHarmonic{ComplexF64}, Inclusion(the 3-dimensional unit sphere), BlockSlice(Block{1, Int64}[Block(1), Block(2)],1:1:4)) with eltype ComplexF64

julia> g = grid(S)
ERROR: LoadError: MethodError: no method matching blockaxes(::QuasiArrays.SubQuasiArray{ComplexF64, 2, SphericalHarmonic{ComplexF64}, Tuple{Inclusion{SphericalCoordinate{Float64}, DomainSets.FixedUnitSphere{SVector{3, Float64}}}, BlockArrays.BlockSlice{BlockRange{1, Tuple{Base.OneTo{Int64}}}, BlockedUnitRange{ArrayLayouts.RangeCumsum{Int64, StepRange{Int64, Int64}}}}}, false}, ::Int64)
Closest candidates are:
  blockaxes(::AbstractArray{T, N}, ::Any) where {T, N} at /home/timon/.julia/packages/BlockArrays/nvJxQ/src/blockaxis.jl:124
  blockaxes(::Any) at /home/timon/.julia/packages/BlockArrays/nvJxQ/src/blockaxis.jl:104
Stacktrace:
 [1] blocksize(A::QuasiArrays.SubQuasiArray{ComplexF64, 2, SphericalHarmonic{ComplexF64}, Tuple{Inclusion{SphericalCoordinate{Float64}, DomainSets.FixedUnitSphere{SVector{3, Float64}}}, BlockArrays.BlockSlice{BlockRange{1, Tuple{Base.OneTo{Int64}}}, BlockedUnitRange{ArrayLayouts.RangeCumsum{Int64, StepRange{Int64, Int64}}}}}, false}, i::Int64)
   @ BlockArrays ~/.julia/packages/BlockArrays/nvJxQ/src/blockaxis.jl:147
 [2] grid(S::QuasiArrays.SubQuasiArray{ComplexF64, 2, SphericalHarmonic{ComplexF64}, Tuple{Inclusion{SphericalCoordinate{Float64}, DomainSets.FixedUnitSphere{SVector{3, Float64}}}, BlockArrays.BlockSlice{BlockRange{1, Tuple{Base.OneTo{Int64}}}, BlockedUnitRange{ArrayLayouts.RangeCumsum{Int64, StepRange{Int64, Int64}}}}}, false})
   @ SphericalHarmonics ~/Documents/Projects/SphericalHarmonics.jl/src/SphericalHarmonics.jl:191
 [3] top-level scope
   @ ~/Documents/Projects/SphericalHarmonics.jl/test/runtests.jl:139
```

Reply:
Ah you have to be on 

https://github.com/JuliaApproximation/OrthogonalPolynomialsQuasi.jl/pull/101

I should figure out a better way of sharing state. Perhaps including a Manifest.toml in a PR until it's ready to merge might help.

Reply:
Okay great, I figured I was on a wrong branch somewhere. 👍 I think having a Manifest in PRs is a good idea.

Reply:
The issue is I don't know how to do this effectively while working with dev paths, which don't include the branch. If I always do `] add ContinuumArrays#mybranch` it will work, but then this won't work with Revise...